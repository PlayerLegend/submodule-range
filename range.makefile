C_PROGRAMS += test/range
C_PROGRAMS += test/buffer
RUN_TESTS += test/range

range-tests: test/range

test/range: src/range/strstr.o
test/range: src/range/range_atozd.o
test/range: src/range/test/range.test.o

tests: range-tests
