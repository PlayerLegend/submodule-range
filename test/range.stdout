set element 0: 5
set element 1: 4
set element 2: 3
set element 3: 2
set element 4: 1
element 0: 5
element 1: 4
element 2: 3
element 3: 2
element 4: 1
